DEVELOPERS.md

```diff
@@ -0,0 +1,140 @@
+# Requirements
+
+You will need the following software installed to complete this setup:
+
+- Homebrew (if you are on macOS), or an equivalent package manager
+- Python 3.10.x (there have been some issues with 3.11)
+- Java 8 or Java 11 (required for running Hail locally)
+
+Since most of our org use Apple laptos for developmnent, this rest of this guide assumes
+that you are operating in a macOS environment.
+
+## Homebrew
+
+Setup homebrew according to the instructions [here](https://brew.sh/). Before closing
```

Comment:
```suggestion
Set up homebrew according to the instructions [here](https://brew.sh/). Before closing
```

Reply:
Thanks! Will likely wait until the hail fix to merge this.